### ISSUE TITLE
Add a configuration file!

### ISSUE THREAD

**Issue by jelly** [archlinux/archlinux-repro#2]
Make it possible to configure the following bash variables:

```
  readonly build_directory=/var/lib/repro
  readonly config_dir=/home/fox/Git/prosjekter/Bash/devtools-repro
  readonly bootstrap_mirror=https://mirror.archlinux.no/iso/latest
```

**Comment by jelly** [comment]
Also make the host_mirror configurable 👍

**Comment by Foxboron** [comment]
Fixed with 42f7ce7

It will check `/etc/devtools-repro/repro.conf` first, then override with `$HOME/.repro.conf` or `$XDG_CONFIG/devtools-repro/repro.conf`. Example is located at `/usr/share/doc/devtools-repro`.
